actuator.c

```diff
@@ -1,94 +1,24 @@
-#include <xc.h>
-#include <stdint.h>
 #include <stdbool.h>
+#include <stdint.h>
+#include <xc.h>
 
-#include "canlib/can.h"
-#include "canlib/can_common.h"
-#include "canlib/pic18f26k83/pic18f26k83_can.h"
-#include "canlib/message_types.h"
-#include "canlib/util/can_tx_buffer.h"
-
-#include "mcc_generated_files/mcc.h"
-
+#include "IOExpanderDriver.h"
 #include "actuator.h"
-#include "board.h"
-#include "timer.h"
+#include "canlib/canlib.h"
 
-void actuator_init(){
-    TRISB5 = 0; // set ACTUATOR_CONTROL (pin 26) as output
-    if (SAFE_STATE == ACTUATOR_OFF) {
-        RELAY_OFF();
-    } else if (SAFE_STATE == ACTUATOR_ON) {
-        RELAY_ON();
-    } else {
-        // Should not get here.
-        while (1) {}; // panic, this will be caught early on in testing
-    }
-    
-#ifdef INJECTOR
-    LATB4 = 1;
-    TRISB4 = 0; // set LIMIT_OPEN as output to power hall sensor
+uint8_t actuator_states = 0;
+uint8_t actuator_invert = 0;
```

Comment:
You should get rid of all the actuator inverting logic. Inverting only makes sense if your CAN commands are OPEN/CLOSE, but our CAN commands are ON/OFF which is well-defined and should never be inverted. This caused a ton of confusion last year, lets pls avoid it this year.